.github/workflows/ci.yml

```diff
@@ -30,16 +30,11 @@ jobs:
   unit-tests:
     name: Unit tests
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        # Run the unit tests both against our oldest supported Python version
-        # and the newest stable.
-        python_version: [ "3.7", "3.x" ]
```

Comment:
It would be very helpful to keep this targetting 3.8 rather than dropping the check for old versions. If there are type hinting problems, we'd be happy to help you through them.